index.html

```diff
@@ -8,18 +8,39 @@
     <title>Document</title>
 </head>
 <body>
-    
-    <h1 class="main-title">Awesome Books</h1>
-    <section class="displayed-books" ></section>
-    <hr class="section-divider">
-    <div class="book-form">
+    <header class="header">
+        <h1 class="header-title">Awesome Books</h1>
+        <ul class="nav">
+            <li class="nav-element border" ><a id="list" class="link" href="#book-list">List</a></li>
+            <li class="nav-element border" ><a id="new" class="link" href="#book-form" >Add new</a></li>
+            <li class="nav-element" ><a id="contact" class="link" href="#contact-info" >Contact</a></li>
+        </ul>
+
+    </header>
+    <h2 class="main-title" id="title-books">All awesome Books</h2>
+    <section class="displayed-books" id="book-list" ></section>
+    <div class="book-form" id="book-form">
         <h2 class="main-title"> Add a new book</h2>
         <label for="title" class="nolabel">title</label>
           <input type="text" class="title-input" id="title" placeholder="title" required> <br>
           <label for="author" class="nolabel">author</label>
           <input type="text" class="author-input" id="author" placeholder="author" required> <br>
         <button class="add-book button"> add</button>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a feature which help you to reset the your inputs filed once a **User** has submitted a form. This will prevent users to submit the same form many times. You can refer this **[article](https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_form_reset)** to help you how to